scripts/migration/migrate_registration_extra.py

```diff
@@ -16,15 +17,38 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+def migrate_file_representation(bad_file):
+    view_url = bad_file.get('viewUrl', '')
+    fid = view_url.split('/')[-2]
```

Comment:
All the data I found on staging has `viewUrl` and IIRC the way we used to do it required that field. I'll change it to `bad_file['viewUrl']`